### ISSUE TITLE
Borders on Tiles?

### ISSUE THREAD

**Issue by SDRanger** [darktrojan/newtabtools#145]
Geoff, I want to open by saying that I have used New Tab Tools for a coupla years now, and adjust and customize it on a near weekly basis to suit my needs, tastes, and moods. WHAT AN EXCELLENT ADD ON!
I have, for some reason, procrastinated in taking a coupla minutes to send you a $ donation.
$$ TONIGHT I DID $$, in the hopes it will at least give my message some attention.The attached screenshots will more easily show the issue that has me climbing the walls. Sadly, on Nov. 9th, the tiles on the screen, (All of which I have spent time creating and customizing…just as it’s developer intended. ) now have the most incredibly unappealing, thick borders around them. Please consider that I always have custom artwork for the huge background image, and all of my tiles are customized as well...the result is a vision of perfect harmony. (Everybody that sees the New Tab Tools screen gives me HUGE compliments, by the way. The word-of-mouth you’re getting is great here!)  My creations are so eye-catching that I have begun taking screen-shots, and it is now very easy to see the amazing-before…and the, well, awful-after. This occurred on Nov 9th, and I don’t recall seeing anything about an update for New Tab Tools…the intrusive, ugly borders just suddenly appeared…literally overnight.
Was this change intentional? I don’t want to come off as over-dramatic here, but every time I click for a new tab and the New Tab Tools screen comes up, I seriously get angry. All that time spent creating and customizing your add-on to suit my taste…as you intended…and now I don’t even want to look at it because it is so ugly.
Is this tile border-phenomenon a glitch on my end?
If this change was intentional, would you ever consider giving us THE OPTION of turning the heavy tile-borders ON or OFF?
Please?
If this is simply how it’s going to look in future versions, is there any way for me to GO BACK to the version we enjoyed up until Nov.8, 2015?
Sorry, I am not a big tecchie here. I didn’t know how to get this message to you other than to create a GitHub account and send this as a “New Issue”.
THANKS for taking the time to read this.
THANKS for creating such an amazing add-on!!
Glenn Burch
San Diego, California, USA
![2015 11 08 new tab tools screenshot](https://cloud.githubusercontent.com/assets/15814436/11116725/49375cce-88e9-11e5-8141-aaad7e50261e.png)
![2015 11 09 new tab tools tiles with borders](https://cloud.githubusercontent.com/assets/15814436/11116726/494bbce6-88e9-11e5-9d00-6b9fe28b01d7.png)

**Comment by darktrojan** [comment]
I've done nothing!

It looks to me as though whatever you're using for your custom theme has decided to make the built-in new tab page prettier. (NTT's page could be mistaken for the built-in one, because in a way, it is it.) I think it's the theme because the brown borders match the toolbars. So have a poke around with it.

It's either that or another add-on interfering. They do that, it's most annoying!

**Comment by SDRanger** [comment]
You're 100% right! It is the Firefox "Walnut" theme, under their "appearances" add-ons. (all of the beige/brown "woodsy" looking stuff across the bottom and all over the top, around the search area and behind all of the tabs.) I deleted it and all is resolved and beautiful again with New Tab Tools.  <sigh....all anger has passed> haha
WHAT a shame though as I've enjoyed Walnut for ages and ages...however, IF I HAVE to make the choice between the two, it would most certainly be New Tab Tools.  You're right, their "update" was MOST annoying. In the future when shopping for a Firefox appearance, I will make sure it accommodates NTTools 100% !   :)
THANKS again for an amazing add-on, and THANKS for taking the time to reply, Geoff!
![ntt all is well again](https://cloud.githubusercontent.com/assets/15814436/11175577/8da1d946-8be8-11e5-930b-4f5e07a9be52.png)
